scss/_modal.scss

```diff
@@ -61,6 +83,14 @@
     height: calc(100vh - (#{$modal-dialog-margin} * 2));
     content: "";
   }
+
+  &.modal-dialog-scrollable {
+    justify-content: center;
+
+    &::before {
+      display: none;
```

Comment:
Wow, you managed to get this working in IE10 without the `::before` hack. Is it possible completely remove the `::before` hack? (maybe we can cover this in another PR if this is the case).

Reply:
Unfortunately, that would not be possible, because scrollable modal does not require flex with `min-height`, but it is necessary for a vertically centered (not scrollable body) modal.

Reply:
That was what I was afraid of. Can you change this to `content: none;` instead of `display: none;`? It would make more sense to have no pseudo element than a hidden pseudo element.